README.md

```diff
@@ -0,0 +1,20 @@
+[![GitHub version](https://badge.fury.io/gh/mellanox%2Flibvma.svg)](https://badge.fury.io/gh/mellanox%2Flibvma)
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/8025/badge.svg)](https://scan.coverity.com/projects/libvma)
+
```

Comment:
@rosenbaumalex Are we guaranteed that forever we'll have converity pass? If we advance in version (and let's say coverity is compromised for some time) - what would happen?